hotel_dump.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python
+"""
+Demonstration of how to download a complete hotel-dump API dump.
+"""
+
+import hashlib
+import logging
+import os.path
+import re
+import sys
+
+import boto3
+
+
+def parse_args():
+    from argparse import ArgumentParser
+    argp = ArgumentParser(__doc__)
+    argp.add_argument("dest_folder", help="Existing empty folder where files should be stored")
+    return argp.parse_args()
+
+
+def md5(path):
+    """
+    Compute MD5 hash of file. Read it in chunks, so we don't need to fix the entire file in memory.
+    :param path: Path of file to hash.
+    :return: Hash value
+    """
+    hash_md5 = hashlib.md5()
+    with open(path, "rb") as f:
+        for chunk in iter(lambda: f.read(4096), b""):
+            hash_md5.update(chunk)
+    return hash_md5.hexdigest()
+
+
+def download_latest(dest_folder):
+    """
+    Download the latest hotel-dump dump into a folder.
+    :param dest_folder: Path to destination folder on local disk.
+    """
+
+    logging.info("Connecting to Amazon S3")
+
+    # Connect to S3
+    s3 = boto3.resource("s3")
+
+    # Select the bucket containing dumps of the TrustYou API.
+    bucket = s3.Bucket("trustyou-api")
+
+    # The trustyou-api bucket contains snapshots of the TrustYou API at different points in time. They are organized in
+    # folders whose name is a timestamp.
+
+    logging.info("Looking for latest complete dump")
+
+    latest_date = None
+    for object_summary in bucket.objects.filter(Prefix="hotels/"):
+        done_file_match = re.match("^hotels/([^/]+)$", object_summary.key)
```

Comment:
the variable name makes no sense, since there is no `done` file as with Meta-Reviews.
Basically what you are checking is that _any_ file exists in the directory and then you are picking the latest one (which is fine and makes sense). Also you could remove the `print` in the next line I think.

Reply:
Hey I will review those and fix that for you